readme.md

```diff
@@ -73,6 +73,12 @@ certificates in the `certs` folder, containing a `rootCA.crt`. Any info you type
 into the prompts is optional. In the next step, you have to register this self-signed
 certificate to your default browser.
 
+#### Using the Proxy-Stack on MacOS
+
+The library `realpath` is mandatory for `DockerExec` to work with local files. You have
+to install the `coreutils` to get that library. With brew installed, use 
+`brew install coreutils`.
```

Comment:
add link to brew installation [Homepage of Homebrew](https://brew.sh/)

Reply:
Thanks! Was added to readme.